vcsutils/utils_test.go

```diff
@@ -148,3 +148,21 @@ func TestGetZeroValue(t *testing.T) {
 	assert.Equal(t, "", GetZeroValue[string]())
 	assert.Equal(t, 0.0, GetZeroValue[float64]())
 }
+
+func TestCreateDotGitFolderWithRemote(t *testing.T) {
+	dir1, err := os.MkdirTemp("", "tmp")
+	assert.NoError(t, err)
+	defer os.RemoveAll(dir1)
+	err = CreateDotGitFolderWithRemote(dir1, "origin", "fakeurl")
+	assert.NoError(t, err)
+	dotGitExist, err := os.Stat(filepath.Join(dir1, ".git"))
+	assert.NoError(t, err)
+	assert.Equal(t, dotGitExist.Name(), ".git")
+	// Return error if .git already exist
+	assert.Error(t, CreateDotGitFolderWithRemote(dir1, "origin", "fakeurl"))
+	// Return error if remote name is empty
+	dir2, err := os.MkdirTemp("", "tmp")
+	assert.NoError(t, err)
+	defer os.RemoveAll(dir2)
+	assert.Error(t, CreateDotGitFolderWithRemote(dir2, "", "fakeurl"))
```

Comment:
I think it is worth putting these lines in a new unit test: TestCreateDotGitFolderWithoutRemote